lambda_backend/route_microservice/src/common/schema.ts

```diff
@@ -5,48 +5,46 @@ const CapitalizedAlphabets = '^[A-Z]*$';
 const NumericDecimalCommaSpace = '^[0-9., ]*$';
 const AsciiCharacters = '^[ -~]+$';
 
+const countryCode = { type: 'string', maxLength: 3, pattern: CapitalizedAlphabets };
+const routeName = { type: 'string', maxLength: 30, pattern: AsciiCharacters };
+const expiredTime = { type: 'string', pattern: ISODateStringPattern };
+const gymLocation = { type: 'string', maxLength: 40, pattern: NumericDecimalCommaSpace };
```

Comment:
Btw is there any other validation for gymLocation? Like making sure it's in the correct format and has valid lat/long?

Reply:
Nope but it should be okay because the only ones inserting to the database is us. The rest just do get request so if they try to put fancy strings, it ll just be rejected